packages/terra-clinical-result/src/common/observation/_Observation.jsx

```diff
@@ -69,7 +65,7 @@ const Observation = (props) => {
     ...customProps
   } = props;
 
-  const isValidValue = !!result.value;
+  const isValidValue = result && !!result.value;
```

Comment:
would optional chainging fulfill this check?
```suggestion
  const isValidValue = result?.value;
```

Reply:
I believe it does, so fixed it here https://github.com/cerner/terra-clinical/pull/648/commits/231120829ac48885ce4397bbe64fff91af5fefd8